examples/cpp/example.cc

```diff
@@ -1,7 +1,7 @@
 /*******************************************************************************
  The MIT License (MIT)
 
- Copyright (c) 2014 Alexander Zolotarev <me@alex.bio> from Minsk, Belarus
+ Copyright (c) 2015 Alexander Zolotarev <me@alex.bio> from Minsk, Belarus
```

Comment:
FTR: Normally, the copyright year is carried over. 2014 is a better choice here compared to 2015.